docs/Passing-Devices-with-Clear-Containers.md

```diff
@@ -0,0 +1,66 @@
+# Using devices with Clear Containers
+
+Host devices can be passed to a container with Docker using --device option
+with docker run. An example of passing a device with docker is given below:
+```
+docker run -it --device=/dev/sdb debian
+```
+Clear Containers has support for passing vfio device groups and block
```

Comment:
s/vfio/VFIO/ ? Also, since this is the first occurence of the term, please can you define it. I see that you're doing that below, so maybe you could restructure slightly to explain what VFIO is first, then go on to explain how to use it with CC?